.ci-tools/vizabi.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+set -e
+
+FORBIDDEN_BRANCHES="master development"
+CURRENT_BRANCH="$(git branch -q --no-color --contains | awk {'print $2'})"
```

Comment:
make be having it working locally is a good idea
but please rewire this script using 
shelljs.local (http://documentup.com/shelljs/shelljs)